pkg/operator/ceph/cluster/cluster.go

```diff
@@ -169,10 +168,11 @@ func (c *ClusterController) initializeCluster(cluster *cluster) error {
 
 	clusterInfo, _, _, err := mon.LoadClusterInfo(c.context, c.OpManagerCtx, cluster.Namespace)
 	if err != nil {
-		logger.Infof("clusterInfo not yet found, must be a new cluster")
+		logger.Info("cluster info not found, must be a new cluster")
 	} else {
 		clusterInfo.OwnerInfo = cluster.ownerInfo
 		clusterInfo.SetName(c.namespacedName.Name)
+		clusterInfo.RequireMsgr2 = mon.RequireMsgr2(cluster.Spec)
```

Comment:
Would it be better to make `RequireMsgr2()` a method of the cluster `Spec` instead? It seems to me that it isn't strictly limited to affecting mons or being determined by mons.

Reply:
Yeah I like that better too